test/unit/org/apache/cassandra/tools/LoaderOptionsTest.java

```diff
@@ -84,38 +84,158 @@ public void testEncryptionSettings() throws Exception
         assertEquals("test.jks", options.clientEncOptions.keystore);
     }
 
+    @Test
+    public void testThrottleDefaultSettings()
+    {
+        LoaderOptions options = LoaderOptions.builder().build();
+        assertEquals(0, options.throttleMebibytes);
+        assertEquals(0, options.interDcThrottleMebibytes);
+    }
+
+    @Test
+    public void testDeprecatedThrottleSettings() throws IOException
+    {
+        // Default Cassandra config
+        File config = new File(Paths.get(".", "test", "conf", "cassandra.yaml").normalize());
+        String[] args = { "-t", "200", "-idct", "400", "-d", "127.9.9.1", "-f", config.absolutePath(), sstableDirName("legacy_sstables", "legacy_ma_simple") };
+        LoaderOptions options = LoaderOptions.builder().parseArgs(args).build();
+        // converts from megabytes to mebibytes
+        assertEquals(24, options.throttleMebibytes);
+        assertEquals(48, options.interDcThrottleMebibytes);
+    }
```

Comment:
After we fix the precision issue, I think it will be correct to check the megabits here